tests/conftest.py

```diff
@@ -16,3 +16,8 @@ def set_seed():
 @pytest.fixture(scope="session", autouse=True)
 def set_default_tensor_type():
     torch.set_default_tensor_type("torch.FloatTensor")
+
+
+@pytest.fixture(scope="function")
+def mcmc_params_testing() -> dict:
+    return dict(num_chains=3, thin=2, warmup_steps=50)
```

Comment:
if we use `num_chains` with `slice_np_vectorized` mostly, I think this should be higher to speed things up, e.g., 10 or 20.

Reply:
does this then run 10 processes or threads? One could look for tests with the `slcie_np_vectorized` sampler and locally overwrite. There were and are quite some overwrites anyway